src/validators/enum_.rs

```diff
@@ -183,8 +185,14 @@ impl EnumValidateValue for PlainEnumValidator {
                         } else if py_input.is_instance_of::<PyFloat>() {
                             return Ok(lookup.validate_int(py, input, false)?.map(|v| v.clone_ref(py)));
                         }
+                        if py_input.is_instance_of::<PyAny>() {
```

Comment:
`is_instance_of::<PyAny>()` I think will always be `true`, will probably be optimized away by the compiler but also not necessary at all IMO.

```suggestion
```